hazelcast-integration/springboot-caching/.gitignore

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
@mustafaiman Local (in folder only) ignore file. Will not affect rest of the projects.